src/lib/api.ts

```diff
@@ -88,7 +87,7 @@ export async function getSuggestions(
 		headers: {
 			'Content-Type': 'application/json'
 		},
-		body: JSON.stringify({ selectedPrompt: selectedPrompt, clear })
+		body: JSON.stringify({ selectedPrompt: selectedPrompt })
```

Comment:
needs to be changed back if we remove the suggestion load in +page.server.ts

Reply:
see above